user-bag-service/src/test/kotlin/io/baum/userbagservice/users/kafka/KafkaConfig.kt

```diff
@@ -0,0 +1,71 @@
+package io.baum.userbagservice.users.kafka
+
+import io.baum.userbagservice.users.kafka.serializer.UserDeserializer
+import io.baum.userbagservice.users.kafka.serializer.UserSerializer
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.apache.kafka.common.serialization.StringSerializer
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.kafka.annotation.EnableKafka
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory
+import org.springframework.kafka.core.*
+import org.springframework.kafka.test.EmbeddedKafkaBroker
+import java.util.concurrent.CountDownLatch
+
+@Configuration
+@EnableKafka
+class KafkaConfig {
+
+    lateinit var received: String
+    lateinit var batchReceived: String
+
+    val latch = CountDownLatch(2)
```

Comment:
How much do you know about CountDownLatches? :) 
Is this a sustainable solution?